document-service/doc-api/pyproject.toml

```diff
@@ -0,0 +1,181 @@
+[tool.poetry]
+name = "doc-api"
+version = "0.1.0"
+description = ""
+authors = ["dlovett <doug@daxiom.com>"]
+license = "Apache 2.0"
```

Comment:
our latest has been using BSD3, so that folks have to attribute where they got it from

Reply:
Updated.